test/Emblem.js

```diff
@@ -45,7 +45,7 @@ describe('Emblem', function(){
         })
     })
     describe('Claimed', ()=>{
-        it('should return false until claimed', ()=>{
+        it('Returns false until claimed', ()=>{
```

Comment:
rename this one to represent that claimed is false when a new emblem is crated

Reply:
done